src/main/java/com/google/sps/data/MockDatastoreManager.java

```diff
@@ -129,4 +141,40 @@ private Hint getHint(int idx) {
         }
         return res.build();
     }
+
+    /**
+    * Returns a random integer in the interval [l,r].
+    * @param l the left endpoint of the interval (inclusive).
+    * @param r the right endpoint of the interval (inclusive).
+    * @return a random integer in [l,r].
+    */
+    private int getRandomIntegerBetween(int l, int r) {
```

Comment:
'l' and 'r' are unclear. Maybe lowerBound and upperBound are clearer variable names.

Reply:
Thanks, renamed to "left" and "right" which is what they're supposed to mean.